app/helpers/audited_helper.rb

```diff
@@ -0,0 +1,10 @@
+module AuditedHelper
+  def enumerize(class_of_item, property, value)
```

Comment:
nit: to me, the name `enumerize` seems to imply that this method will take a value and turn it into an enumeration, or something like that?  I think that this method does the opposite- takes a value which might be an item in an enumeration, and turns it into a string (looked up in localized strings)?  Maybe a name more like `localize_enum`, or `t_enum` or something would be more descriptive?